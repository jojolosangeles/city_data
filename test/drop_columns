set -x
python ../code/main.py scripts/drop_columns.txt > actual/drop_columns.out
diff expected/drop_columns.out actual/drop_columns.out
