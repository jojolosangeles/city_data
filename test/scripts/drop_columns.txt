sample20 = data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv
sample20.columns
sample20.drop DR Number
sample20.columns
sample20.drop Date Reported, Area ID
sample20.columns
