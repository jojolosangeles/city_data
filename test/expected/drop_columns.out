Test: scripts/drop_columns.txt
> load data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv as sample20
Loaded data/sample_20_Traffic_Collision_Data_from_2010_to_Present.csv into DataFrame sample20
> columns sample20
['DR Number' 'Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID'
 'Area Name' 'Reporting District' 'Crime Code' 'Crime Code Description'
 'MO Codes' 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
> drop sample20 DR Number
Dropping ['DR Number'] from DataFrame 'sample20'
Columns dropped
> columns sample20
['Date Reported' 'Date Occurred' 'Time Occurred' 'Area ID' 'Area Name'
 'Reporting District' 'Crime Code' 'Crime Code Description' 'MO Codes'
 'Victim Age' 'Victim Sex' 'Victim Descent' 'Premise Code'
 'Premise Description' 'Address' 'Cross Street' 'Location']
> drop sample20 Date Reported, Area ID
Dropping ['Date Reported', 'Area ID'] from DataFrame 'sample20'
Columns dropped
> columns sample20
['Date Occurred' 'Time Occurred' 'Area Name' 'Reporting District'
 'Crime Code' 'Crime Code Description' 'MO Codes' 'Victim Age'
 'Victim Sex' 'Victim Descent' 'Premise Code' 'Premise Description'
 'Address' 'Cross Street' 'Location']
Test completed: scripts/drop_columns.txt
